addons/web/views/base_document_layout_views.xml

```diff
@@ -32,6 +32,7 @@
                             <field name="company_details" string="Address" options="{'resizable': false}"/>
                             <field name="report_header" string="Tagline" placeholder="e.g. Global Business Solutions" options="{'resizable': false}"/>
                             <field name="report_footer" placeholder="Write your phone, email, bank account, tax ID, ..." string="Footer" options="{'resizable': false}"/>
+                            <field name="paperformat_id" widget="selection" required="1" domain="[('report_ids', '=', False)]"/>
```

Comment:
Why this domain?

Reply:
It's needed to remove the paperformat that are linked to a report, and so if we remove the domain we have paperformat that we don't want in the configuration layout 
![image](https://github.com/odoo/odoo/assets/109144554/62b47e61-bee9-4af0-bd8d-8d0be0c58a49)